wca-regulations.md

```diff
@@ -88,7 +88,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - 2i2b) The competitor must not interact with (e.g. operate, hold, wear) any active camera. Exception: the competitor may wear a camera mounted on their head, as long as it is out of their sight and it is clear that they are not interacting with it (apart from wearing it).
 - 2j) The WCA Delegate may disqualify a competitor from a specific event.
     - 2j1) If a competitor is disqualified from an event for any reason, they are not eligible for any more attempts in the event.
-        - 2j1a) The results of all remaining attempts in the event are recorded as DNF.
+        - 2j1a) If the competitor had already started at least one attempt in a round before being disqualified, the results of all remaining attempts in the event are recorded as DNF. If the competitor had started no attempts, no results should be recorded.
```

Comment:
One small question before merging this PR:

> If the competitor had started no attempts, no results _should_ be recorded.

What is the purpose of this "should"? Wouldn't it be enough to phrase is similar to the previous sentence?

> If the competitor had started no attempts, no results _are recorded_.

Reply:
You're right! I'll push another commit fixing this.